endpoints/_includes/functions.php

```diff
@@ -51,6 +51,8 @@
     'toll_province_bias' => [ 'description' => '' , 'default' => null, 'overridable' => true, 'hidden' => false],
     'toll_free_province_bias' => [ 'description' => '' , 'default' => '', 'overridable' => true, 'hidden' => false],
     'tomato_helpline_routing' => [ 'description' => '', 'default' => false, 'overridable' => true, 'hidden' => false],
+	'tomato_helpline_fallback' => [ 'description' => '', 'default' => false, 'overridable' => true, 'hidden' => false],
```

Comment:
formatting issues with spaces needs to be fixed

Reply:
Possible tab vs space difference as noted above - they were otherwise cut/pasted from tomato_helpline_routing and just had the text changed for the setting.

Reply:
Was using notepad++ for quick editing - it was saving in a format it was not displaying. Everything looked well aligned on there until I looked at it with other editors. Used the basic notepad as a catch all since it has no formatting ability and everything looked aligned in there now.